core/templates/services/audio-player.service.ts

```diff
@@ -189,6 +188,19 @@ export class AudioPlayerService {
     }
   }
 
+  getProgress(): number {
+    if (this.getAudioDuration() > 0) {
+      return this.getCurrentTime() / this.getAudioDuration();
+    }
+    return 0;
+  }
+
+  setProgress(progress: number): void {
+    if (progress > 0 && progress < 1) {
```

Comment:
why can't we have progress 0 or 1? the getProgress can return bot 0 and 1 so I was expecting setProgress to accept both of these values.

Reply:
done